scripts/pop_sim.py

```diff
@@ -1,16 +1,15 @@
 from neurophotonics.sim import Tissue, Fluorescence, Detection
+from scripts.helper import keys_used
 
-try:
-    Tissue.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("Tissue already populated")
-
-try:
-    Detection.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("Detection already populated")
-
-try:
-    Fluorescence.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("Fluorescence already populated")
+print(
+    "keys processed:",
+    keys_used(
+        Tissue.populate(reserve_jobs=True, display_progress=False, processes=1024)
+    )
+    + keys_used(
+        Detection.populate(reserve_jobs=True, display_progress=False, processes=1024)
+    )
+    + keys_used(
+        Fluorescence.populate(reserve_jobs=True, display_progress=False, processes=1024)
+    ),
```

Comment:
These are RAM-heavy processes, so I would be careful with multiprocessing.

Reply:
The problem is that these are RAM-hungry processes that use multithreaded linear algebra libraries. It's unlikely that performance improves with `processes > 2` but the risk of running out of RAM and bogging down with swaps is high.

Reply:
In that case, I will set it to 1 for now and test later if setting processes to 2 yields an appreciable improvement.

Reply:
My guess is that `processes=2` will likely yield peak performance while still being relatively safe on RAM.

Reply:
Alright, I have set processes to 2 for now, though we will likely have to tweak this value in the future as we further automate the workflow.